src/views/MyRequests/Table/index.tsx

```diff
@@ -43,40 +44,10 @@ import {
 } from 'utils/tableSort';
 import user from 'utils/user';
 
-import { AppState } from '../../../reducers/rootReducer';
-
 import { MergedRequestsForTable } from './mergedRequestForTable';
 
 import '../index.scss';
 
-const calcSystemIntakeNextMeetingDate = (
```

Comment:
@adamodd @ClayBenson94 related change - this is not feature flagged but does use new fields from BE